docs/local_source_upload.md

```diff
@@ -17,21 +17,22 @@ shp build create sample-nodejs \
     --output-image="docker.io/<namespace>/sample-nodejs:latest"
 ```
 
-2. Clone the repository or use a pre-existing one. For example:
```

Comment:
nit: please leave the original numbering sequence, otherwise GitHub renders the markdown only having `1.` entries.

Reply:
Thanks for the hint. Actually, I wanted to fix a markdown linter issue. Normally, we should only use `1.` and let the renderer do the counting.

<img width="579" alt="image" src="https://user-images.githubusercontent.com/1979133/163213350-358a8763-41ac-423c-8035-a5b39acf9332.png">

The actual issue it seems is the intend of the code blocks. They are not aligned with the respective numbered entry. If one fixes that, the numbering works with just `1.`, which makes the linter happy and the right numbering. Commit is coming.

Reply:
I also quickly fixed the other markdown linter warnings in the file.